BlueprintUI/Sources/Element/ElementState.swift

```diff
@@ -436,13 +456,37 @@ final class ElementState {
             return nodes
         }
 
-
         /// We already have a cached layout, reuse that.
         if let existing = layouts[size] {
             delegate.ifDebug {
                 $0.treeDidReturnCachedLayout(existing, for: self)
             }
 
+            if hasUpdatedChildrenDuringLayout == false {
```

Comment:
@kdaker Wanted to make sure you saw this – it's a new addition to ensure that even when returning cached layouts, we're still doing a lightweight enumeration of the tree to update to the latest `element`. This is important, eg, if the backing view description changes or needs to be consistently re-applied, but `isEquivalent` did not change.

Reply:
(So far in perf testing, there's basically little to no cost to this; though I might be missing something)